js/packages.js

```diff
@@ -1,254 +1,282 @@
-let allPackages, currentPackages, cancellationToken, hiddenCount;
-const triples = ["arm-uwp","arm64-windows","x64-linux","x64-osx","x64-uwp","x64-windows","x64-windows-static","x86-windows"];
-let compatFilter = [];
+let allPackages, currentPackages, cancellationToken, hiddenCount
+const triples = [
+    'arm-uwp',
+    'arm64-windows',
+    'x64-linux',
+    'x64-osx',
+    'x64-uwp',
+    'x64-windows',
+    'x64-windows-static',
+    'x86-windows',
+]
+let compatFilter = []
 
 var getUrlParameter = function getUrlParameter(sParam) {
     var sPageURL = window.location.search.substring(1),
         sURLVariables = sPageURL.split('&'),
         sParameterName,
-        i;
+        i
```

Comment:
Could you change this to four `var`s?

```suggestion
    var sPageURL = window.location.search.substring(1)
    var sURLVariables = sPageURL.split('&')
    var sParameterName
    var i
```